src/public/pages/scripts/components/emojiPicker.css

```
@@ -6,6 +6,18 @@
 	max-height: 360px;
 }
 
+.emoji-picker>.sr-only {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	padding: 0;
+	margin: -1px;
+	overflow: hidden;
+	clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)